lib/tracker/mode_s/crc_realworld_integration_test.go

```diff
@@ -0,0 +1,256 @@
+package mode_s_test
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"plane.watch/lib/producer"
+	"plane.watch/lib/tracker/beast"
+	"plane.watch/lib/tracker/mode_s"
+)
+
+// TestCRCWithRealWorldData validates CRC implementation against real Beast format data
+func TestCRCWithRealWorldData(t *testing.T) {
+	// Read the beast file
+	data, err := os.ReadFile("cmd/stress/full-feed.beast")
```

Comment:
please place testdata in a testdata/ folder in the same dir (lib/tracker/mode_s/testdata)